ci-operator/config/openshift-priv/ovn-kubernetes/openshift-priv-ovn-kubernetes-release-4.18.yaml

```
@@ -401,8 +401,8 @@ tests:
         OVN_LOCAL_GATEWAY_MODE=true
         FEATURE_SET=TechPreviewNoUpgrade
     workflow: baremetalds-e2e
-- as: e2e-vsphere-ovn-techpreview
-  optional: true
+- always_run: true
```

Maybe we don't even need to change the priv jobs ? 
```suggestion
- always_run: false
```